### PR TITLE
Hide recent detailed log output from libmachine

### LINKED ISSUE #5278: 1.4.0-beta.0: machine driver logs showing to stdout
I feel like this behavior changed within the last day or so, but I'm not sure which PR did so:

```
% minikube start
😄  minikube v1.4.0-beta.0 on Debian 
🔥  Creating kvm2 VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...
Running pre-create checks...
Creating machine...
(minikube) Creating KVM machine...
(minikube) Setting up store path in /usr/local/google/home/tstromberg/.minikube/machines/minikube ...
(minikube) Building disk image from file:///usr/local/google/home/tstromberg/.minikube/cache/iso/minikube-v1.4.0-beta.0.iso
(minikube) Downloading /usr/local/google/home/tstromberg/.minikube/cache/boot2docker.iso from file:///usr/local/google/home/tstromberg/.minikube/cache/iso/minikube-v1.4.0-beta.0.iso...
(minikube) Setting executable bit set on /usr/local/google/home/tstromberg/.minikube/machines/minikube (perms=drwx------)
(minikube) Setting executable bit set on /usr/local/google/home/tstromberg/.minikube/machines (perms=drwxr-xr-x)
(minikube) Setting executable bit set on /usr/local/google/home/tstromberg/.minikube (perms=drwxr-xr-x)
(minikube) Setting executable bit set on /usr/local/google/home/tstromberg (perms=drwx--x--x)
(minikube) Creating domain...
(minikube) Creating network...
(minikube) Ensuring networks are active...
(minikube) Ensuring network default is active
(minikube) Ensuring network minikube-net is active
(minikube) Getting domain xml...
(minikube) Creating domain...
(minikube) Waiting to get IP...
(minikube) Found IP for machine: 192.168.39.83
(minikube) Waiting for SSH to be available...
(minikube) KVM machine creation complete!
Waiting for machine to be running, this may take a few minutes...
Detecting operating system of created instance...
Waiting for SSH to be available...
Detecting the provisioner...
Provisioning with buildroot...
Setting Docker configuration on the remote daemon...
Checking connection to Docker...
Docker is up and running!
Waiting for SSH to be available...
Detecting the provisioner...
🐳  Preparing Kubernetes v1.16.0-beta.1 on Docker 18.09.8 ...
```

### PR DESCRIPTION
**Pull request by afbjorklund**
Fixes #5278 (while still fixing #5114)

### CODE REVIEW

File: cmd/minikube/cmd/root.go [hunk 1]
```diff
@@ -87,6 +92,10 @@ var RootCmd = &cobra.Command{
 				exit.WithError("logdir set failed", err)
 			}
 		}
+
+		if enableUpdateNotification {
+			notify.MaybePrintUpdateTextFromGithub()
```
Comment (tstromberg) [inline cmd/minikube/cmd/root.go]: I feel that upgrade notices should only be part of "minikube start". It makes little sense to pester users elsewhere, IMHO. What do you think?

Comment (afbjorklund) [reply cmd/minikube/cmd/root.go]: No idea, I just reverted the failing commit

But I normally disable the warning, so yes

**Comment by afbjorklund** [comment]
Sorry, just fixing the issues seen on `master`. Some kind of "glog" magic happening in notify ?

**Comment by tstromberg** [comment]
Obsoleted by #5344

**Pull request closed without merging**